docker/README.md

```diff
@@ -0,0 +1,3 @@
+Usage
+
+```docker build --no-cache=true -f docker/ubuntu/Dockerfile .```
```

Comment:
I would tag the image too by adding `-t parity`. This makes the output of `docker images` a bit more friendly:

```
❯ docker images
REPOSITORY                     TAG                 IMAGE ID            CREATED             SIZE
parity                         latest              9f8e2b065950        7 minutes ago       1.74 GB
```